hicommonwealth/commonwealth
# Created generic `<CardSlider/>` component

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/7916

## Description of Changes
Created generic `<CardSlider/>` component for `<AdminOnboardingSlider/>` and `<UserTrainingSlider/>`.

## "How We Fixed It"
N/A

## Test Plan
1. Create a community
2. Visit any community page
3. Verify that `<AdminOnboardingSlider/>` works correctly as before
4. Now create a new account
5. Verify you see the `<UserTrainingSlider/>` and it works correctly as before

## Deployment Plan
N/A

## Other Considerations
N/A
